History.md

```diff
@@ -32,182 +27,153 @@
   * Rack::Test now generates no Ruby warnings
 
-== 0.4.2 / 2009-09-01
+## 0.4.2 / 2009-09-01
 
 * Minor enhancements
-
   * Merge in rack/master's build_multipart method which covers additional cases
   * Accept raw :params string input and merge it with the query string
   * Stringify and upcase request method (e.g. :post => "POST") (Josh Peek)
-
 * Bug fixes
-
   * Properly convert hashes with nil values (e.g. :foo => nil becomes simply "foo", not "foo=")
   * Prepend a slash to the URI path if it doesn't start with one (Josh Peek)
   * Requiring Rack-Test never modifies the Ruby load path anymore (Josh Peek)
   * Fixed using multiple cookies in a string on Ruby 1.8 (Tuomas Kareinen and Hermanni Hyytiälä)
 
-== 0.4.1 / 2009-08-06
+## 0.4.1 / 2009-08-06
 
 * Minor enhancements
-
-  * Support initializing a Rack::Test::Session with an app in addition to
-    a Rack::MockSession
+  * Support initializing a `Rack::Test::Session` with an app in addition to
+    a `Rack::MockSession`
   * Allow CONTENT_TYPE to be specified in the env and not overwritten when
     sending a POST or PUT
 
-== 0.4.0 / 2009-06-25
+## 0.4.0 / 2009-06-25
 
 * Minor enhancements
-
-  * Expose hook for building Rack::MockSessions for frameworks that need
+  * Expose hook for building `Rack::MockSessions` for frameworks that need
     to configure them before use
   * Support passing in arrays of raw cookies in addition to a newline
     separated string
   * Support after_request callbacks in MockSession for things like running
     background jobs
   * Allow multiple named sessions using with_session
-  * Initialize Rack::Test::Sessions with Rack::MockSessions instead of apps.
+  * Initialize `Rack::Test::Sessions` with `Rack::MockSessions` instead of apps.
     This change should help integration with other Ruby web frameworks
     (like Merb).
   * Support sending bodies for PUT requests (Larry Diehl)
 
-== 0.3.0 / 2009-05-17
+## 0.3.0 / 2009-05-17
 
 * Major enhancements
-
   * Ruby 1.9 compatible (Simon Rozet, Michael Fellinger)
-
 * Minor enhancements
-
-  * Add CookieJar#[] and CookieJar#[]= methods
+  * Add `CookieJar#[]` and `CookieJar#[]=` methods
   * Make the default host configurable
-  * Use Rack::Lint and fix errors (Simon Rozet)
-  * Extract Rack::MockSession from Rack::Test::Session to handle tracking
+  * Use `Rack::Lint` and fix errors (Simon Rozet)
+  * Extract `Rack::MockSession` from `Rack::Test::Session` to handle tracking
     the last request and response and the cookie jar
   * Add #set_cookie and #clear_cookies methods
   * Rename #authorize to #basic_authorize (#authorize remains as an alias)
     (Simon Rozet)
 
-== 0.2.0 / 2009-04-26
+## 0.2.0 / 2009-04-26
 
-Because #last_response is now a MockResponse instead of a Rack::Response,
-#last_response.body now returns a string instead of an array.
+Because #last_response is now a MockResponse instead of a Rack::Response, #last_response.body
+now returns a string instead of an array.
```

Comment:
jepp, i did it first but wass't sure if it would have enough value. I will update the PR accordingly. Thanks for the feedback.